WpfApp5/WpfApp5/MainWindow.xaml

```diff
@@ -0,0 +1,57 @@
+﻿<Window x:Class="WpfApp5.MainWindow"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:local="clr-namespace:WpfApp5"
+        xmlns:view="clr-namespace:WpfApp5.View"
+        xmlns:viewModel="clr-namespace:WpfApp5.ViewModel"
+        mc:Ignorable="d"
+        Title="MainWindow" Height="850" Width="800">
+    <Window.DataContext>
+        <viewModel:ToysViewModel/>
+    </Window.DataContext>
+    <Window.Resources>
+        <DataTemplate x:Key="toysView">
+            <view:ToysView/>
+        </DataTemplate>
+        <DataTemplate x:Key="toysFullView">
+            <view:ToysFullView/>
+        </DataTemplate>
+    </Window.Resources>
+    <Grid>
```

Comment:
I have a lot of questions to the layout. It looks pretty weird and can't be demonstrated to anyone.
We should make layout as beautiful, user-friendly as possible.

Reply:
-1 to layout.

Reply:
Sorry, but the layout is not flexible. Try to stretch the window.

Reply:
-1 to layout.